### DISCUSSION TITLE
UDM-Pro update 1.12.30 -> 2.x

### DISCUSSION THREAD

**Discussion by michaelblaas** [fabianishere/udm-iptv#181] [General]
Hi,

The releasenotes of Unifi-OS version 1.12.30 states:

_An upcoming version of UDM/UDM-Pro will migrate to UniFi OS 2.x. The process is currently being well-tested internally and will be released shortly, with no specific ETA._

What can be expected of this update in terms of the correct functioning of IPTV after updating? Is 2.x the same OS as the SE and do we have to deal with the limitation of not being able to persist the settings? Or is a complete new install needed?

Regards,
Michael

**Comment by fabianishere** [comment]
I don’t known for certain until the new firmware is released for the UDM (Pro). However , I expect the process to be the same as the UDM SE.
